app/code/BelodubrovskyiAn/Lesson7Module/view/frontend/layout/knockout_index_index.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<page layout='1column' xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd" >
+    <head>
+        <title>
```

Comment:
In your case the string with title will contain new line characters before and after the actual title. Please rename to 
```
<title>Knock Out Testing</title>
```

Reply:
Fixed